pkg/config/load.go

```
@@ -112,3 +113,36 @@ func OperateOnCIOperatorConfigDir(configDir string, callback func(*cioperatorapi
 		return nil
 	})
 }
+
+func LoggerForInfo(repoInfo FilePathElements) *logrus.Entry {
+	return logrus.WithFields(logrus.Fields{
+		"org":         repoInfo.Org,
+		"repo":        repoInfo.Repo,
+		"branch":      repoInfo.Branch,
+		"source-file": repoInfo.Filename,
+	})
+}
+
+type Info struct {
+	Configuration cioperatorapi.ReleaseBuildConfiguration
+	RepoInfo      FilePathElements
+}
+
+func (i *Info) Logger() *logrus.Entry {
+	return LoggerForInfo(i.RepoInfo)
+}
+
+func (i *Info) CommitTo(dir string) {
```

I think this should propagate the errors and not eat them